src/test/java/hudson/plugins/jira/UpdaterTest.java

```diff
@@ -297,7 +297,7 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
         );
 
         // expected issue list
-        final List<JiraIssue> expectedIssuesToCarryOver = new ArrayList<JiraIssue>();
+        final Set<JiraIssue> expectedIssuesToCarryOver = Sets.newHashSet();
```

Comment:
Sets.newLinkedHashSet(); ? If running JDK 8+ order as stated in the javadocs is not predictable and can cause random test assertion failures.